template/nuxt-app/assets/vars/colors.json

```diff
@@ -2,5 +2,7 @@
 	"primary-color": "#009c4b",
 	"primary-color-dark": "#008c44",
 	"secondary-color": "#e8179e",
-	"offwhite": "#edf0f5"
+	"offwhite": "#edf0f5",
+	"ui-color": "#e0e0e0",
```

Comment:
What would `ui-color` be used for?

Reply:
I've used that naming convention for things like dividers, borders around certain elements, etc. But I can remove it, and we can just make it project by project

Reply:
No, i like that idea, maybe just a different name then?  Maybe just like `ui-border-color` (since `border-color` is a css property)?

Reply:
sounds good, re-added and called it `ui-border-color`. If this gets merged in I'll add it to my swatches